tools/ci_build/github/linux/docker/manylinux/requirements.txt

```diff
@@ -0,0 +1,6 @@
+numpy==1.16.6
```

Comment:
naive question but why are there different requirements.txt in the first place. why is some software like mypy required here but not in other places.

Reply:
Different container may have different requirements. The "requests" package was brought in for onnxruntime server, and it has special license requirement. So, I'd prefer to have different requirements.txt for such cases. 
mypy is needed for building onnx from source I guess.